app/MMB/src/scenes/tripServices/transportation/TransportationMap.js

```diff
@@ -138,21 +138,22 @@ export class TransportationMap extends React.Component<Props, State> {
     }
   }
 
-  openLocationPicker = async () => {
-    const { currentLocation } = this.state;
-    let currentAddress;
-
-    if (currentLocation != null) {
-      const { latitude, longitude } = currentLocation;
-      currentAddress = await this.getFormattedAddress({
-        latitude,
-        longitude,
-      });
-    }
-
-    this.props.navigation.navigate('AddressPickerScreen', {
-      currentAddress,
-    });
+  openLocationPicker = () => {
+    navigator.geolocation.getCurrentPosition(
```

Comment:
we have a geolaction context, maybe you could use that one instead?

Reply:
Ah, of course! And actually, I just saw in that context file
```
// TODO: Show alert to tell user to go to settings an allow us to access location
// Need alert with translations first
```
which I can do in this PR too 😄